bricks/_common/arm_none_eabi.mk

```diff
@@ -52,7 +52,7 @@ $(error failed)
 endif
 endif
 endif
-ifeq ($(PB_LIB_BTSTACK),1)
+ifeq ($(or $(PB_LIB_BTSTACK),$(PB_LIB_BTSTACK_CLASSIC)),1)
```

Comment:
It is still the same library, so I would just give it a new value instead of adding another variable.

e.g. `PB_LIB_BTSTACK = 1` is BLE only `PB_LIB_BTSTACK = 2` is classic only (and perhaps in the future `PB_LIB_BTSTACK = 3` is dual mode).

Or just leave it as-is and let it compile a few extra files even if they aren't used.

Reply:
We could use strings instead of numbers. And use the `$(filter pattern…,text)` make function to do the matching.